### ISSUE TITLE
Unable to set the width of the combobox

### ISSUE THREAD

**Issue by PavithraNarayanan18** [SAP/fundamental-ngx#12545]
#### Is this a bug, enhancement, or feature request?

Bug

#### Describe your proposal.

Unable to set the width of the combobox https://sap.github.io/fundamental-ngx/#/core/combobox

#### Can you handle that on the application side

No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)

Angular - 17.2.1
Fundamental-ngx - 0.50.2

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;

For combobox we were not able to set the width of the element. There is no any attribute to set the width of the combobox in ngx nor we were not able to add width to combobox using external css

Component used : https://sap.github.io/fundamental-ngx/#/core/combobox

#### Please provide relevant source code (if applicable).

<div fd-form-item>
        <label fd-form-label for="comboboxID1">Standard</label>
        <fd-combobox
            inputId="comboboxID1"
            ariaLabel="Standard"
            maxHeight="250px"
            placeholder="Type some text..."
            [dropdownValues]="fruits"
            [(ngModel)]="searchTermOne"
        >
        </fd-combobox>
    </div>

#### Please provide stackblitz example(s).

https://stackblitz.com/edit/q14fbt?file=src%2Fapp%2Fcombobox-example.component.html,src%2Fapp%2Fcombobox-example.component.scss

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.

NA

#### Did you check the documentation and the API?

Yes

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by InnaAtanasova** [comment]
hi @PavithraNarayanan18 , thanks for reporting the problem. Could you check if the fix I provided [here](https://github.com/SAP/fundamental-ngx/pull/12594) would be sufficient for your case. Thanks

**Comment by PavithraNarayanan18** [comment]
Hi @InnaAtanasova, can we able to give the width in percentage (Eg: Width:10%) so that it is dynamic when screen size changes

**Comment by InnaAtanasova** [comment]
> Hi [@InnaAtanasova](https://github.com/InnaAtanasova), can we able to give the width in percentage (Eg: Width:10%) so that it is dynamic when screen size changes

yes, it is possible to use percentage. I added an example to the documentation https://github.com/SAP/fundamental-ngx/pull/12984
